src/tcgwars/logic/impl/gen4/Platinum.groovy

```diff
@@ -1009,11 +1009,11 @@ public enum Platinum implements LogicCardInfo {
               checkLastTurn()
               checkNoSPC()
               assert my.hand : "Your hand is empty"
-              assert my.all.find{it.numberOfDamageCounters} : "Your Pokémon are healthy"
+              assert my.all.find {it.numberOfDamageCounters} : "Your Pokémon are healthy"
```

Comment:
> Q. Can I use Blissey's "Nurse Call" Poke-POWER to discard a card from my hand even if I have no damaged Pokemon in play?
>
> A. Yes, you can. (Jun 11, 2009 PUI Rules Team)

This shouldn't be asserted at all (heal below could be replaced with a filer, then doing the heal only if valid targets are found.